packages/video_player/video_player_avfoundation/ios/Classes/Cache/Cache/CacheAction.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSUInteger, CacheActionType) { CacheActionTypeLocal = 0, CacheActionTypeRemote };
```

Comment:
local and remote dont sound like "actions". maybe just cache type? also what does remote cache mean?

Reply:
Remote and local cache types are used for optimisation purposes. You basically can use parts of existing local cache's regardless of the validity. From objective-c documentation: `No
    existing local cache data, regardless of its freshness or validity,
    should be used to satisfy a URL load request.`

Reply:
i meant what are local cache and remote cache? can you link me to any documentation?